### PR TITLE
SASS-296 - Added get help link to Layout.html

### PR DESCRIPTION
**Pull request by Dula-J**

### CODE REVIEW

File: app/views/headerFooterTemplate/Layout.scala.html [hunk 1]
```diff
@@ -17,13 +17,15 @@
...
+@import views.html.templates.helpers.Link
 
 @this(
         govukLayout: GovukLayout,
         govukHeader: GovukHeader,
         head: Head,
         beforeBodyContent: BeforeBodyContent,
-        hmrcStandardFooter: HmrcStandardFooter
+        hmrcStandardFooter: HmrcStandardFooter,
+        link: Link,
```
Comment (joeyymoore) [inline app/views/headerFooterTemplate/Layout.scala.html:28]: Not sure if you want this to be named getHelpLink: Link incase we have to add links in the future so you know which link it is?

File: test/views/helpers/LinkViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  private val aTagSelector = "a"
+
+  "LinkView" when {
+    val link = "/contact/some-system"
+    val message = "Get help with this page"
+    val id = Some("tag-id")
+    val isExternal = true
+    val opensInNewTab = "(opens in new tab)"
+
+    "provided with " should {
```
Comment (joeyymoore) [inline test/views/helpers/LinkViewSpec.scala:37]: might be me being stupid, but provided with what?🙈

File: app/views/headerFooterTemplate/Layout.scala.html (hunk 1 above)
Comment (Dula-J) [reply app/views/headerFooterTemplate/Layout.scala.html:28]: Will make the change.

File: test/views/helpers/LinkViewSpec.scala (hunk 2 above)
Comment (Dula-J) [reply test/views/helpers/LinkViewSpec.scala]: You're not, it does not read well. I've made a change, let me know what you think.

**Pull request merged**